components/NavBar/NavBar.tsx

```diff
@@ -105,58 +94,48 @@ export const NavBar = () => {
   }
 
   return (
-    <>
-      <UserNavbarContainer>
-        <UserNavbar aria-label={intl.formatMessage({ id: "navbar.ariaUserMenu" })}>
-          <Cart />
-          {/* Hiding unused icons
-          <Button href="/tasks" variant="ghost" color="secondary">
-            <FormattedMessage id="navbar.taskStatistic" values={{ remaining: <span>0</span> }} />
+    <HeaderContentContainer>
+      <LogoContainer>
+        <NextLink href="/" passHref>
+          <Button as="a" variant="ghost" color="secondary">
+            <Typography><FormattedMessage id="navbar.logo" /></Typography>
           </Button>
-          <Button variant="ghost_icon" color="secondary">
-            <Icon data={refresh} title={intl.formatMessage({ id: "navbar.titleRefresh" })} />
-          </Button>  */}
-          <Button variant="ghost_icon" color="secondary">
-            <Icon data={account_circle} title={intl.formatMessage({ id: "navbar.titleAccount" })} />
-          </Button>
-        </UserNavbar>
-      </UserNavbarContainer>
-
-      <Header>
-        <HeaderContentContainer>
-          <LogoContainer>
-            <NextLink href="/" passHref>
-              <Button as="a" variant="ghost" color="secondary">
-                <Typography><FormattedMessage id="navbar.logo" /></Typography>
-              </Button>
-            </NextLink>
-          </LogoContainer>
-
-          <nav aria-label="Main navigation">
-            <ActionsContainer>
-              <Link href="/browse">
+        </NextLink>
+      </LogoContainer>
+      {/*  <Link href="/browse">
                 <Button variant="ghost" color="secondary">
                   <Icon data={explore} title={intl.formatMessage({ id: "navbar.browse" })} />
                   <FormattedMessage id="navbar.browse" />
                 </Button>
               </Link>
-              {/*
               <Link href="/browse">
                 <Button variant="ghost" color="secondary">
                   <Icon data={comment_discussion}
                    title={intl.formatMessage({ id: "navbar.community" })} />
                   <FormattedMessage id="navbar.community" />
                 </Button>
               </Link>
-              */}
-            </ActionsContainer>
-          </nav>
-
-          <SearchForm onSubmit={onSearchSubmit}>
-            <Search aria-label="sitewide" id="search-normal" placeholder={intl.formatMessage({ id: "navbar.placeholderSearch" })} onChange={onSearchChange} value={searchQuery} />
-          </SearchForm>
-        </HeaderContentContainer>
-      </Header>
-    </>
+  */}
+
+      <ActionsContainer>
```

Comment:
According to the [EDS docs](https://eds-storybook-react.azurewebsites.net/?path=/docs/navigation-topbar--introduction#with-search-and-icons), the search field should be in the CustomContent and not the Actions container